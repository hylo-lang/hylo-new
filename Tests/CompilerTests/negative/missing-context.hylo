//! no-std

struct A<T> { public memberwise init }

fun g<T, U>(_ x: T) {}

fun main() {
  _ = A()
  g(())
}
