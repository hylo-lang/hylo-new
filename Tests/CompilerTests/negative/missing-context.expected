negative/missing-context.hylo:8.7-10: error: not enough context to infer a type
  _ = A()
      ~~~
negative/missing-context.hylo:9.3-4: error: not enough context to infer a type
  g(())
  ^
